__tests__/components/editor/GroupChoiceModal.test.js

```diff
@@ -52,24 +50,28 @@ describe('<GroupChoiceModal />', () => {
       expect(wrapper.find(Modal.Body).find('form').find('select').length).toEqual(1)
     })
 
-    it('has the first select option as "PCC"', () => {
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .first()
-        .childAt(0)
-        .text()).toEqual('PCC')
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .first()
-        .prop('value')).toEqual('pcc')
+    it('has the first select option as "Cornell University"', () => {
+      const firstOpt = wrapper.find(Modal.Body).find('form select option').first()
+      expect(firstOpt.text()).toEqual('Cornell University')
+      expect(firstOpt.prop('value')).toEqual('cornell')
     })
 
-    it('has the last select option as "University of Washington"', () => {
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .last()
-        .childAt(0)
-        .text()).toEqual('University of Washington')
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .last()
-        .prop('value')).toEqual('wau')
+    it('has the last select option as "Yale University"', () => {
+      const lastOpt = wrapper.find(Modal.Body).find('form select option').last()
+      expect(lastOpt.text()).toEqual('Yale University')
+      expect(lastOpt.prop('value')).toEqual('yale')
+    })
```

Comment:
my first instinct was to get rid of this test and the one above, in light of the one below, which implicitly tests what these two test.  but... i couldn't quite get myself to do it, because i like the simplicity/directness of these two.  hopefully i at least made them a little more readable?

happy to ditch if people would prefer.